topi/python/topi/math.py

```diff
@@ -408,3 +416,57 @@ def reinterpret(x, dtype):
         The result.
     """
     return cpp.reinterpret(x, dtype)
+
+
+@tvm.tag_scope(tag=tag.ELEMWISE)
+def logical_and(lhs, rhs):
+    """Compute element-wise logical and of data.
+        Parameters
```

Comment:
for the op description part, better to keep similar indentation as the rest.  https://github.com/dmlc/tvm/blob/master/python/tvm/relay/op/tensor.py#L421-L454

Reply:
logical_not also be needed to add to tvm, because doc has this op

Reply:
Hi @yongwww @gyshi
I complete the suggestion you mentioned, please help to review it, thanks.